plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_ar.properties

```diff
@@ -0,0 +1,90 @@
+command.org.jkiss.dbeaver.erd.diagram.view.description     = \u0645\u0634\u0627\u0647\u062F\u0629 \u0645\u062E\u0637\u0637 \u0631\u0648\u0627\u0628\u0637 \u0627\u0644\u0643\u064A\u0627\u0646\u0627\u062A (ERD)
+command.org.jkiss.dbeaver.erd.diagram.view.name            = \u0645\u0634\u0627\u0647\u062F\u0629 \u0627\u0644\u0645\u062E\u0637\u0637
+command.org.jkiss.dbeaver.erd.focus.diagram.description    = \u062A\u063A\u064A\u064A\u0631 \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0625\u0644\u0649 \u0627\u0644\u0645\u062E\u0637\u0637
+command.org.jkiss.dbeaver.erd.focus.diagram.name           = ERD: \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0639\u0644\u0649 \u0627\u0644\u0645\u062D\u0637\u0637
+command.org.jkiss.dbeaver.erd.focus.outline.description    = \u062A\u063A\u064A\u064A\u0631 \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0639\u0644\u0649 \u0627\u0644\u062D\u062F\u0648\u062F 
+command.org.jkiss.dbeaver.erd.focus.outline.name           = ERD: \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0639\u0644\u0649 \u0627\u0644\u062D\u062F\u0648\u062F
+command.org.jkiss.dbeaver.erd.focus.palette.description    = \u062A\u063A\u064A\u064A\u0631 \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0639\u0644\u0649 \u0627\u0644\u0644\u0648\u062D\u0629
+command.org.jkiss.dbeaver.erd.focus.palette.name           = ERD: \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0639\u0644\u0649 \u0627\u0644\u0644\u0648\u062D\u0629
+command.org.jkiss.dbeaver.erd.focus.parameters.description = \u062A\u063A\u064A\u064A\u0631 \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0639\u0644\u0649 \u0627\u0644\u0645\u062F\u062E\u0644\u0627\u062A 
+command.org.jkiss.dbeaver.erd.focus.parameters.name        = ERD: \u0627\u0644\u062A\u0631\u0643\u064A\u0632 \u0639\u0644\u0649 \u0627\u0644\u0645\u062F\u062E\u0644\u0627\u062A
+command.org.jkiss.dbeaver.erd.toggleHand.description       = \u0627\u0644\u062A\u0646\u0642\u0644 \u0628\u064A\u0646 \u0623\u062F\u0627\u0629 \u0627\u0644\u064A\u062F \u0648\u0627\u0644\u0623\u062F\u0627\u0629 \u0627\u0644\u0645\u0633\u062A\u062E\u062F\u0645\u0629 \u0633\u0627\u0628\u0642\u064B\u0627
+command.org.jkiss.dbeaver.erd.toggleHand.name              = \u062A\u0628\u062F\u064A\u0644 \u0623\u062F\u0627\u0629 \u0627\u0644\u064A\u062F
+
+confirm.erd_open_exported_file.message       = \u0647\u0644 \u062A\u0631\u063A\u0628 \u0628\u0641\u062A\u062D \u0627\u0644\u0645\u0644\u0641 \u0627\u0644\u0645\u0635\u062F\u0631 \u0641\u064A \u0627\u0644\u0628\u0631\u0646\u0627\u0645\u062C \u0627\u0644\u0645\u0631\u0628\u0648\u0637
+confirm.erd_open_exported_file.title         = \u0641\u062A\u062D \u0627\u0644\u0645\u0644\u0641 \u0627\u0644\u0645\u0635\u062F\u0631
+confirm.erd_open_exported_file.toggleMessage = \u0644\u0627 \u062A\u0633\u0623\u0644\u0646\u064A \u0645\u062C\u062F\u062F\u064B\u0627
+
+editor.erd.description                                          = \ \u0645\u062E\u0637\u0637 \u0631\u0648\u0627\u0628\u0637 \u0627\u0644\u0643\u064A\u0627\u0646\u0627\u062A (ERD)
```

Comment:
I noticed that some sentences start with backslashes followed by a space (here and below). Could you please clarify if this is intentional? If not, would you mind removing them? Thank you!

Reply:
Since Arabic is read right to left, I assume those are spaces. They need to be escaped to avoid being trimmed at runtime.